SPECS/jna/ant_tools_arbitrary_ow2.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/build-ant-tools.xml b/build-ant-tools.xml
+index bb73fc74..b0e93f3d 100644
+--- a/build-ant-tools.xml
++++ b/build-ant-tools.xml
+@@ -4,7 +4,7 @@
+     <property name="build" value="build" />
+     <target name="default">
+         <mkdir dir="${build}/ant-tools" />
+-        <javac classpath="lib/ant.jar:lib/asm-8.0.1.jar"
++        <javac classpath="lib/ant.jar:lib/objectweb-asm_asm.jar"
```

Comment:
Callout: currently we build objectweb 7.2 and coerce it into this spot where 8.0.1 is expected... objectweb-asm is not a trivial upgrade as the Sources schema was complex and they seem to have been moved somewhere else.